packages/pluggableWidgets/slider-web/src/Slider.editorConfig.ts

```diff
@@ -0,0 +1,136 @@
+import { hidePropertiesIn, hidePropertyIn, Problem, Properties } from "@mendix/piw-utils-internal";
+import { MaxValueTypeEnum, MinValueTypeEnum, SliderPreviewProps, StepSizeTypeEnum } from "../typings/SliderProps";
+
+const keysToHideByMinValueType: Record<MinValueTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["minAttribute", "expressionMinimumValue"],
+    dynamic: ["staticMinimumValue", "expressionMinimumValue"],
+    expression: ["staticMinimumValue", "minAttribute"]
+};
+
+const keysToHideByMaxValueType: Record<MaxValueTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["maxAttribute", "expressionMaximumValue"],
+    dynamic: ["staticMaximumValue", "expressionMaximumValue"],
+    expression: ["staticMaximumValue", "maxAttribute"]
+};
+
+const keysToHideBySizeType: Record<StepSizeTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["stepAttribute", "expressionStepSize"],
+    dynamic: ["stepValue", "expressionStepSize"],
+    expression: ["stepValue", "stepAttribute"]
+};
+
+export function getProperties(values: SliderPreviewProps, defaultProperties: Properties): Properties {
+    hidePropertiesIn(defaultProperties, values, keysToHideByMinValueType[values.minValueType]);
+
+    hidePropertiesIn(defaultProperties, values, keysToHideByMaxValueType[values.maxValueType]);
+
+    hidePropertiesIn(defaultProperties, values, keysToHideBySizeType[values.stepSizeType]);
+
+    if (!values.showTooltip) {
+        hidePropertyIn(defaultProperties, values, "tooltip");
+    }
+
```

Comment:
@diego-antonelli don't we need the following here as well?:
```
if (platform === "web") {
        transformGroupsIntoTabs(defaultProperties);
    }
```

Reply:
Probably it will come with PM Review from Studio, but good to implement already

Reply:
As it was implemented I am resolving this discussion